### DISCUSSION TITLE
Is it possible to upgrade/update Clover from windows or a Live CD?

### DISCUSSION THREAD

**Discussion by Iondefeso** [CloverHackyColor/CloverBootloader#540] [Q&A]
Hi all, firstly I hope everyone is doing great, and I would like to start by stating that I don't have macOS or any interest to install it anytime soon. What I do have is Windows 11 and Fedora Linux 36 both installed and encrypted in a RAID 0 (SSDs). Now I use clover simply as a bootloader that boots directly from my EFI partition. Why? the windows bootloader doesn't see Linux and grub2 doesn't see Windows. I heard you can program the grub somehow but that is too complicated for me at my current knowledge level so I followed a youtube guide to install clover from windows mounting the EFI partition and all the rest.. that fixed the original problem.

The new problem is that the clover I currently have is very old (Matrix UI) and I've seen the newer ones and and they look way more sophisticated. So I wonder if is possible to upgrade my clover installation by downloading the latest clover .zip or .iso and just replacing the files the same way I installed it from windows? Or if there is any LiveCD I can boot to upgrade? 

I couldn't find anything on my searches so decided to make an account here to ask. Sorry for the long post and thank you.

**Comment by Morbius01** [comment]
I am using Clover to boot Windows 11 and Kali Linux. See my post [here](https://github.com/CloverHackyColor/CloverBootloader/discussions/491).

Could you just clarify a few points ?

You said that you currently use an old version of Clover. Can you boot both W11 and Fedora from Clover ?

If it is the case, you just want to upgrade to a more recent version of Clover to benefit from its latest features (wondering which ones...), you could create a new Clover USB key, and see if you can boot your OSs from it.

If it is not the case, then you have another problem.

Which version of Clover (old) do you currently use ?
Which version of Clover would you like to upgrade too ?
How old is your mother board ? Is it EFI compatible, or do you use a legacy BIOS ?
